plugins/DNS/requirements.txt

```diff
@@ -0,0 +1,2 @@
+dnspython @ git+https://github.com/filips123/dnspython@dns-over-https
```

Comment:
Please hard-code commit hash for security.

Reply:
@imachug Done!